app/src/main/java/com/stevesoltys/seedvault/ui/recoverycode/RecoveryCodeInputFragment.kt

```diff
@@ -105,8 +103,6 @@ class RecoveryCodeInputFragment : Fragment() {
 
         if (viewModel.isRestore) {
             introText.setText(R.string.recovery_code_input_intro)
-            backView.visibility = VISIBLE
-            backView.setOnClickListener { requireActivity().finishAfterTransition() }
```

Comment:
Why was the back view removed here? Is the back functionality provided in a different way now?